rows/plugins/xls.py

```diff
@@ -18,6 +18,7 @@
 from __future__ import unicode_literals
 
 import datetime
+import StringIO
```

Comment:
The `StringIO` module is gone on Python 3. We do not support it yet but using `io.BytesIO` or `io.StringIO` now will make the migration easier. Could you please change it?